boards/arm/reel_board/doc/reel_board.rst

```diff
@@ -0,0 +1,434 @@
+.. _reel_board:
+
+reel board
+##########
+
+Overview
+********
+
+reel board is a evaluation board based on the Nordic Semiconductor
+nRF52840 SoC. The board was developed by PHYTEC Messtechnik GmbH in
+cooperation with Zephyr |trade| Project for the Hackathon - "Get Connected".
+The board has also a built-in debug adapter based on the DAPLink interface
+firmware and NXP MK20DX128VFM5 SoC.
+
+reel board is more than just another evaluation board.
+It is equipped with the Electrophoretic Display and environmental sensors.
+This not only makes it easy to try out Zephyr OS, but the following
+use cases can also be evaluated:
+
+* battery powered sensor node
+* low power, low cost HMI for remote control or environmental sensor monitoring
+* interactive badge for meetings and conferences
+* product, name or price tag
+* just a temperature and humidity sensor on your table
+
+Zephyr applications use the reel_board board configuration
+to run on the reel board hardware. It provides
+support for the Nordic Semiconductor nRF52840 ARM |reg| Cortex |reg|-M4F SoC
+with integrated 2.4 GHz transceiver supporting Bluetooth |reg| Low Energy
+and IEEE |reg| 802.15.4.
+
+Hardware
+********
+
+On the front of the board are RGB-LED, ADPS9960 and HDC1010 sensors,
+Electrophoretic Display and odd pins from the edge connector.
+The RGB-LED is controlled by the nRF52840 via GPIO pins.
+Display is controlled by the nRF52840 via SPI and 3 GPIOs.
+
+On the back side of the board are all other components such as nRF52840,
+circuit of Debug Adapter, On/Off and power source switch, battery holder,
+buttons and MMA8652 sensor.
+
+ADPS9960 is Digital Proximity, Ambient Light, RGB and Gesture sensor.
+HDC1010 is a digital humidity and temperature sensor.
+MMA8652 is a 12-bit Digital Accelerometer.
+All sensors are connected to the I2C bus and one GPIO pin each,
+which can be used as an interrupt source.
+
+.. figure:: img/reel_board_descr_front.png
+     :width: 442px
+     :align: center
+     :alt: reel board front
+
+     reel board front (Credit: PHYTEC)
+
+.. figure:: img/reel_board_descr_back.png
+     :width: 442px
+     :align: center
+     :alt: reel board back
+
+     reel board back (Credit: PHYTEC)
+
+Power supply
+============
+
+The board is optimized for low power applications and supports two
+power source configurations, battery and micro USB connector.
+
+The On/Off switch can choose which power source is used.
+
+reel board uses TPS610981 boost converter to generate supply voltage
+for nRF52840 and peripherals (sensors and EPD).
+The boost converter has two modes:
+
+* Active mode - supply voltage for nRF52840 and peripherals are on
+* Low Power mode - only supply voltage for nRF52840 is on
+
+The mode is controlled by MODE pin (P1.00).
+
+.. note::
+   Actually there is no possibility to reduce energy consumption by the
```

Comment:
This is confusing.  The previous bullet points said peripheral power is off in low-power mode?

Reply:
Yes, but it can not be used by the software and is not yet implemented.